### PR TITLE
DEVPROD-11945 Fix cloning test 

### PR DESCRIPTION
**Pull request by malikchaya2**
DEVPROD-11945

### CODE REVIEW

File: operations/fetch_test.go [hunk 1]
```diff
@@ -5,63 +5,35 @@ import (
...
+		owner:      "evergreen-ci",
+		repository: "sample",
+		revision:   "cf46076567e4949f9fc68e0634139d4ac495c89b",
+		branch:     "main",
 	}
+	opts.rootDir = t.TempDir()
+	assert.NoError(t, clone(opts))
 
-	testCases := map[string]testCase{
-		"SimpleHTTPS": {isPassing: true, opts: cloneOptions{
```
Comment (ablack12) [inline operations/fetch_test.go:22]: Are we able to keep this format and just remove the SimpleHTTPS case? Lmk if I'm missing something about what was changed here.

Comment (malikchaya2) [reply operations/fetch_test.go:22]: the simpleHTTPS case is still there, it's just the first setup
https://github.com/evergreen-ci/evergreen/pull/8386/files#diff-f3e98349a5ccd629b2b8e1c1c2603cbfe2b619824af2caefccee4b1a270062ecR12-R16

**Review by ablack12 — COMMENTED** [review]
I'm going to mark myself as reviewed re our DM conversation about maintaining isolated test cases.

File: operations/fetch_test.go [hunk 2]
```diff
@@ -46,15 +42,25 @@ func TestClone(t *testing.T) {
 		}},
 	}
 
+	opts := cloneOptions{
```
Comment (ablack12) [inline operations/fetch_test.go:45]: Can opts be defined in the for loop? This maintains the pattern of consistency / not requiring tests to run in a particular order.

**Review by ablack12 — APPROVED** [review]
LGTM with last comment

File: operations/fetch_test.go (hunk 2 above)
Comment (malikchaya2) [reply operations/fetch_test.go:45]: I tried that but moving the opts definition into the for loop causes the test to fail :(

**Pull request merged**